docs/en/stack/ml/anomaly-detection/ml-ad-finding-anomalies.asciidoc

```diff
@@ -69,13 +69,8 @@ visualizations that are customized to help you analyze your data.
 [[ml-ad-setup]]
 == 2. Set up the environment
 
-If you want to use {ml-features}, there must be at least one {ml} node in
-your cluster and all master-eligible nodes must have {ml} enabled. For more
-information about these settings, see 
-{ref}/modules-node.html#ml-node[{ml} nodes].
-
-If {stack-security-features} are enabled, you must also ensure your users have
-the necessary privileges. See <<setup>>.
+If you want to use {ml-features}, refer to the <<setup>> page for the required 
```

Comment:
I think that since they're on this page, we already know they want to use ML features, so I'd suggest omitting that qualifier. For example:
```suggestion
Refer to the <<setup>> page for the required 
```